src/article_metrics/tests/test_api_v1.py

```diff
@@ -17,8 +19,18 @@ def tearDown(self):
     def test_monthly_data(self):
         self.assertEqual(0, models.Article.objects.count())
         self.assertEqual(0, models.Metric.objects.count())
-        month_to_import = datetime(year=2015, month=8, day=0o1)
-        logic.import_ga_metrics('monthly', from_date=month_to_import, to_date=month_to_import)
+        month_to_import = datetime(year=2015, month=8, day=1)
+
+        def test_output_path(result_type, from_date, to_date):
+            # ignore whatever dates given, return path to fixture
+            if result_type == 'views':
+                return join(self.fixture_dir, 'test_import_ga_monthly_stats', 'views', '2015-08-01_2015-08-31.json')
+            if result_type == 'downloads':
+                return join(self.fixture_dir, 'test_import_ga_monthly_stats', 'downloads', '2015-08-01_2015-08-31.json')
```

Comment:
should there be a `raise SomeException` at the end of `test_output_path` in case `result_type` is an unforeseen value?

Reply:
I disabled it for some reason a year ago: 
https://github.com/elifesciences/elife-metrics/blame/develop/src/article_metrics/ga_metrics/core.py#L170

I'll reinstate it.

Reply:
turns out the new non-article stuff now relies on it so these tests just raise a value error if something unexpected is encountered, which it won't